File: app/src/main/java/com/yhjoo/dochef/model/ExpandTitle.java
package com.yhjoo.dochef.model;

import com.chad.library.adapter.base.entity.AbstractExpandableItem;
import com.chad.library.adapter.base.entity.MultiItemEntity;

public class ExpandTitle extends AbstractExpandableItem<ExpandContents> implements MultiItemEntity {
    public String title;

    public ExpandTitle(String title) {
        this.title = title;
    }

    @Override
    public int getItemType() {
        return 0;
    }

    @Override
    public int getLevel() {
        return 0;
    }
}

File: app/src/main/java/com/yhjoo/dochef/data/model/ExpandTitle.kt
package com.yhjoo.dochef.data.model

import com.chad.library.adapter.base.entity.AbstractExpandableItem
import com.chad.library.adapter.base.entity.MultiItemEntity

class ExpandTitle(var title: String?) : AbstractExpandableItem<ExpandContents?>(), MultiItemEntity {
    override fun getItemType(): Int {
        return 0
    }

    override fun getLevel(): Int {
        return 0
    }
}